projects/igniteui-angular/src/lib/grids/hierarchical-grid/hierarchical-grid.component.ts

```diff
@@ -561,6 +561,26 @@ export class IgxHierarchicalGridComponent extends IgxHierarchicalGridBaseDirecti
         return this._defaultExpandState;
     }
 
+    /**
+     * Gets/Sets the entities used for advanced filtering.
+     *
+     * @remarks
+     * This property is required in remote data scenarios.
+     * @example
+     * ```typescript
+     * const remoteEntities = this.grid.remoteEntities;
+     * this.grid.remoteEntities = [];
+     * ```
+     */
+    @Input()
+    public set filteringEntities(entities: EntityType[]) {
```

Comment:
Why not just `entities`? I mean someone can use it in a non-filtering scenario & it's shorter :)

Reply:
In the current implementation these entities are only used for hierarchical grid in advanced filtering. What non-filtering scenario do you have in mind?

Reply:
True, but this is a public input of the HierarchicalGrid component, so customers could make use of it in a different scenario, like idk, doing some formatting even.